### PR TITLE
Problem:(Fix #42) Added more instructions and reference to chain-maind cli page in the testnet docs

### LINKED ISSUE #42: Problem: No un-jail instruction in the testnet docs
It is provided in the [chain-maind cli page](https://chain.crypto.com/docs/wallets/cli.html#tx-slashing-unjail-unjailing-a-validator) but not included in the testnet docs.

### PR DESCRIPTION
**Pull request by crypto-matto**
Instead of adding un-jail instruction, added instructions of transfer, staking & query balance. A note is also added at the end for redirecting people to chain-maind cli page.

### CODE REVIEW

**Review by calvinaco — APPROVED** [review]
LGTM, Although the title and the PR content seems not match? haha

**Comment by crypto-matto** [comment]
Oh cos I was thinking to just copy the title of the issue #42 . I'll be mindful next time.

File: docs/getting-started/croeseid-testnet.md [hunk 1]
```diff
@@ -261,8 +261,27 @@ Alternatively, you can run it on this [browser based IDE](https://repl.it/@allth
 $ cat ~/.chain-maind_testnet/config/priv_validator_key.json | jq -r '.pub_key.value'
 ```
 
-## Step 4. Transactions subcommands
-### Step 4-1. `tx bank send ` - Transfer operation
+## Step 4. Perform Transactions
+### Step 4-1. `query bank balances` - Check your transferable balance
+
+You can check your _transferable_ balance with the `balance` command under the bank module.
```
Comment (lezzokafka) [inline docs/getting-started/croeseid-testnet.md:267]:
```suggestion
You can check your _transferable_ balance with the `balances ` command under the bank module.
```

File: docs/getting-started/croeseid-testnet.md [hunk 2]
```diff
@@ -280,7 +299,49 @@ confirm transaction before signing and broadcasting [y/N]: y
...
+confirm transaction before signing and broadcasting [y/N]: y
+```
+
+:::
+
+#### **Unbond your delegated funds** [`tx staking unbond <validator-addr> <amount>`]
+
+On the other hand, we can create a `Unbond` transaction to unbond the delegated funds
+
+::: details Example: Unbond funds from a staking address
```
Comment (lezzokafka) [inline docs/getting-started/croeseid-testnet.md:325]:
```suggestion
::: details Example: Unbond funds from a validator under the address `tcrocncl16k...edcer`
```

File: docs/getting-started/croeseid-testnet.md [hunk 3]
```diff
@@ -280,7 +299,49 @@ confirm transaction before signing and broadcasting [y/N]: y
 
 :::
 
+### Step 4-3. `tx staking ` - Staking operations
+
+Staking operations involve the interaction between an address and a validator. It allows you to create a validator and lock/unlocking funds for staking purposes.
+
+#### **Delegate you funds to a validator** [`tx staking delegate <validator-addr> <amount>`]
+
+To bond funds for staking, you can deposit funds (an unspent transaction) to a _staking_ address by the `Deposit` operation.
```
Comment (lezzokafka) [inline docs/getting-started/croeseid-testnet.md:308]:
```suggestion
To bond funds for staking, you can delegate funds  to a validator by the `delegate` command
```

Sorry I think there is some  thaler concept left in here: 
https://github.com/crypto-com/chain-docs/blob/master/docs/wallets/cli.md#delegate-you-funds-to-a-validator-tx-staking-delegate-validator-addr-amount

**Review by lezzokafka — APPROVED** [review]
Cheers @crypto-matto  :D !

**Pull request merged**